Comp2024/src/main/java/frc/robot/Constants.java

```diff
@@ -88,6 +90,23 @@ public static final class Ports
     public static final int    kDIO1_NoteInFeeder    = 1;
   }
 
+  /////////////////////////////////////////////////////////////////////////////

```

Comment:
These path constants should have the correct naming applied with a preceding "k" like the others in the file. Also, if you skim through the Constants.java file, you'll see that there is a (mostly) intentional order with all of the joystick items, subsystems, then autos and vision at the bottom. It would be nice to have these with the rest of the limelight constants.